e2e_tests/test_suites/guestpolicies/guest_policies_utils.go

```diff
@@ -277,3 +277,101 @@ done`
 		Value: &ss,
 	}
 }
+
+func getRecipeInstallStartupScript(image, recipeName, pkgManager string) *computeApi.MetadataItems {
```

Comment:
nit: s/Install/IsInstalled

currently it sounds like it prepares the startup script that installs recipe